tox.ini

```diff
@@ -0,0 +1,35 @@
+[tox]
+envlist = report-ruff, report-darglint, report-pylint
+requires = 
+    ruff
+    darglint
+    pylint
+    black
+
+[testenv:format-black]
+deps =
+    black
+commands =
+    black -l 100 event-driven-ansible/extensions/eda/plugins
```

Comment:
The path needs to be updated to `/extensions/eda/plugins` for all the linters. The path included the collection name only when it was outside of the root dir.